hypha/apply/projects/templates/application_projects/invoice_detail.html

```diff
@@ -16,12 +16,31 @@ <h5 class="heading heading--no-margin">{% trans "For" %}: {{ object.project.titl
 <div class="wrapper wrapper--sidebar wrapper--outer-space-medium">
     <div class="wrapper--sidebar--inner">
         <div class="card card--solid">
-            <p class="card__text"><b>{% trans "Status" %}:</b> {{ object.get_status_display }}</p>
+            <p class="card__text"><b>{% trans "Invoice Number" %}:</b> {{ object.vendor_document_number }}</p>
```

Comment:
```suggestion
            <p class="card__text"><b>{% trans "Invoice No." %}:</b> {{ object.vendor_document_number }}</p>
```

Keep it short!? :) cc: @fourthletter